.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, this is introducing a new top-level project.  Did you mean to do that @saulhidalgoaular ?

Reply:
This is a good question. I deleted it many times, but the Eclipse was generating it again automatically. I saw this behaviour for first time in Eclipse 4.22 (current update). At the end, I decided to push it, so it will not be generated again. I will dedicate tonight some time to research in detail what is the idea of this file.

Reply:
In a Slack conversation we agreed to include the new `.project` file.